tests/js/index.js

```diff
@@ -66,7 +66,6 @@ if (global.enableSyncTests) {
     TESTS.SetSyncTests = node_require("./set-sync-tests");
     TESTS.DictionarySyncTests = node_require("./dictionary-sync-tests");
     TESTS.MixedSyncTests = node_require("./mixed-sync-tests");
-    TESTS.AnalyticsTests = require("./analytics-tests");
```

Comment:
Do you think it's worth having some kind of integration test for the new way of doing analytics? Maybe one for a follow up in any case

Reply:
See my comment above about the staging.  If we don't have access to such a bucket, I don't think we can meaningfully do integration tests on this.

Reply:
I think it could still be interesting to have, for example, a set of skeleton "test projects" with the various different Podfile/package.json permutations that we know of (RN Hermes iOS, RN Hermes Android, RN JSC iOS, RN JSC Android, Electron etc...), then run the script on them with a mocked `https` library and assert that (parts of) the payload match what we expect for each project.

Reply:
I agree - having tests for this would be great.

Reply:
At the moment, we will skip it all if `CI` is set - which Github Actions does.